tests/phpunit/bootstrap.php

```diff
@@ -1,7 +1,7 @@
 <?php
 $_tests_dir = getenv( 'WP_TESTS_DIR' );
 if ( ! $_tests_dir ) {
-	$_tests_dir = '/tmp/wordpress-tests-lib';
+	$_tests_dir = '/var/folders/91/8p8tzc_111xfs4zxljt06s6m0000gn/T/wordpress-tests-lib';
```

Comment:
Will this directory name always be the same?
`8p8tzc_111xfs4zxljt06s6m0000gn`

Reply:
Probably not - likely this was accidentally pushed.

Reply:
Replying before I undo the commit: Yes, this was accidentally pushed. The `$TMPDIR` variable that is used by the WooCommerce Tests Installer does not default to `/tmp/` on my machine and I had to change it.